config/_shared.json

```diff
@@ -132,5 +132,6 @@
 	"enable_all_sections": true,
 	"sections": {
 		"jetpack-cloud": false
-	}
+	},
+	"theme": "default"
```

Comment:
Do we really need to introduce yet another concept for determining the visual appearance at a framework level? And we already have color schemes, which the user selects - how do those play together with the new concept of themes?

Reply:
Currently this doesn't do anything but I could see that in the future this might be a thing of its own. Should we set it to false instead?

Reply:
In general, I'd recommend `null` instead of `false` because `false` could set the expectations that a `true` is also an available value. Note that this will need a change where this config value is handled.

Just to keep it simple, I'll reconsider and suggest that you leave it as it is. We can always change it later.